src/biz/bokhorst/xprivacy/PrivacyService.java

```diff
@@ -2293,24 +2291,7 @@ private boolean isAMLocked(int uid) {
 	}
 
 	private Context getContext() {
-		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
```

Comment:
I had left this, because I don't like to use privacy variable for compatibility reasons, but since this is now been done for Lollipop, you are right this can go away.